pages/about.html

```diff
@@ -0,0 +1,101 @@
+            </ul>
+            <button class="d-btn">cc campaign</button>
+        </nav>
+        <ul class="mobile-nav">
+            <li><a href="../index.html">Home</a></li>
+             <li><a href="#program">Program</a></li>
+             <li><a href="#join">Join</a></li>
+             <li><a href="#sponsor">Sponsor</a></li>
+             <li><a href="#news">News</a></li>
+     </ul>
+     <div class="times">&times</div>
+    </header>
+    <section class="about-hero">
+        <p class="p-1">"Hello! Dev World"</p>
+        <h1 class="h-1">YOUNG DEV'S GLOBAL SUMMIT UGANDA</h1>
+        <p class="p-2">The Young Dev Summit brings together the community of  young,expert devs.
+            this year's summit is aimed to spreading the love of tech to the young generation
+            With the help of our partners
+         </p>
+         <p class="p-3">
+            Please contact us per Email for further questions about Young Dev's Global Summit
+         </p>
+         <a class="p-4" href="#">haylederrick14@gmail.com</a>
+    </section>
+    <section class="about-logo">
+        <div class="logo-header">
+            <h2>Young Dev's Global Summit Uganda Logo</h2>
+            <div class="line"></div>
+            <p class="logo-paragraph">The logo of young dev's global summit was decided through the logo competition</p>
+        </div>
+        
+        <a class="logo-image" href="#"><span class="img-span">Y-</span>Dev</a>
+    </section>
+    <section class="past-summits">
+        <div class="past-header">
+            <h2>See the past Global Summits</h2>
+            <div class="line"></div>
+            <p>Take a look at the past two Global summits that took place in kampala, uganda</p>
+        </div>
+        <div class="past-container">
+            <div class="card-1">
+                <h3>2013</h3>
+                <p>yd Global Summit in kampala,Uganda</p>
+            </div>
+            <div class="card-2">
+                <h3>2011</h3>
+                <p>Yd Global Summit in Nairobi,Kenya</p>
+            </div>
+        </div>
+    </section>
+    <section class="partners">
+        <div class="partner-header">
+            <h2>partners</h2>
+            <div class="line"></div>
+        </div>
+        <div class="partner-container">
+            <img class="partner-img" src="/assets/airbnb.png" alt="airbnb">
+            <img class="partner-img" src="/assets/google.png" alt="google">
+            <img class="partner-img" src="/assets/meta.png" alt="meta">
+            <img class="partner-img" src="/assets/microsoft.png" alt="microsoft">
+            <img class="partner-img" src="/assets/mozilla.webp" alt="mozilla">
+        </div>
+    </section>
+    <footer>
+        <img class="footer-img" src="/assets/cc-logo.png" alt="logo">
+        <div class="foooter-div">
+            <p>2023 Young Devs Uganda</p>
+            <span>All Rights Reserved</span>
+        </div>
+    </footer>
```

Comment:
- [ ] Footer of the about page shoul be dark in the background and white in the text color to match the given design:
your design:
![image](https://user-images.githubusercontent.com/91486791/234959431-50aafd04-240e-4fed-a7b8-5ff9da335a78.png)
actual:
![image](https://user-images.githubusercontent.com/91486791/234959528-fbfe7f48-4334-46bc-8318-8d20c645f404.png)